### PR TITLE
Fix issue #151, dng-dnm assumes DP is zero and improve BCF parsing er…

### PR DESCRIPTION
**Pull request by jgarciamesa**
…ror messages

### CODE REVIEW

**Comment by reedacartwright** [comment]
Jenkins, okay to test.

File: src/include/dng/task/dnm.h [hunk 1]
```diff
@@ -47,6 +47,12 @@ inline void add_app_args(po::options_description &desc, arg_t &arg) {
 #undef XM
     ;
 }
+// define macros for BCF parsing error handling
+#define missing_PL -6
+#define non_existent_ALT -11
+#define missing_member_1 -1
+#define missing_member_2 -2
+#define missing_member_3 -3
```
Comment (stevenhwu) [inline src/include/dng/task/dnm.h]: Maybe use UPPER_CASE for define?
#define MISSING_PL
#define MISSING_MEMBER_3

Comment (jgarciamesa) [reply src/include/dng/task/dnm.h]: Yes, I will change that.

**Comment by reedacartwright** [comment]
Can you add the DNG_ prefix before your definitions?

**Pull request merged**